From a145ea12918e9c7f2bdbca855b2719e3c7acf51b Mon Sep 17 00:00:00 2001
From: Pirama Arumuga Nainar <pirama@google.com>
Date: Fri, 9 Nov 2018 11:53:41 -0800
Subject: [PATCH 3/5] Move 8.0.3.profdata -> 8.0.4.profdata

Bug: http://b/119270185

Reuse existing profdata since 8.0.4 just has a cherry-pick on top of
8.0.3.

Test: build.py
Change-Id: I9dc6ae49e95ace2b60c6bda4d8550e65b1deefd3
---
 profiles/{8.0.3.profdata => 8.0.4.profdata} | Bin
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename profiles/{8.0.3.profdata => 8.0.4.profdata} (100%)

diff --git a/profiles/8.0.3.profdata b/profiles/8.0.4.profdata
similarity index 100%
rename from profiles/8.0.3.profdata
rename to profiles/8.0.4.profdata
-- 
2.17.1

